recipes/cloud-data-connector/description.md

```diff
@@ -0,0 +1,37 @@
+Cloud Data Connector is a tool to connect to AzureML, Azure blob, GCP storage, GCP Big Query and AWS storage S3. The goal is provide all cloud managers in one place and provide documentation for an easy integration.
```

Comment:
This files doesn't seem to be necessary. Is it?

Reply:
I agree with you, thanks